app/Filament/Resources/ServerResource/Pages/EditServer.php

```diff
@@ -659,7 +677,57 @@ public function form(Form $form): Form
                                     ->deletable(false)
                                     ->addable(false)
                                     ->columnSpan(4),
-                            ])->columns(4),
+                                Forms\Components\Actions::make([
+                                    Action::make('createDatabase')
+                                        ->authorize(fn (Database $database) => auth()->user()->can('create database', $database))
```

Comment:
Only `view`, `update` and `delete` take the model as 2nd param.

```suggestion
                                        ->authorize(fn () => auth()->user()->can('create database'))
```